### PR TITLE
feat: support gop_autogen jump

### PR DESCRIPTION
**Pull request by xhyqaq**

### CODE REVIEW

File: internal/godoc/render.go [hunk 1]
```diff
@@ -151,6 +152,10 @@ func (p *Package) renderOptions(innerPath string, sourceInfo *source.Info, modIn
 		if p.Line == 0 { // invalid Position
 			return ""
 		}
+		// resolve gop_autogen.gop path
+		if strings.Contains(p.Filename, ".gop") || strings.Contains(p.Filename, ".gox") {
```
Comment (xushiwei) [inline internal/godoc/render.go]: don't need check file ext

File: internal/godoc/render.go [hunk 2]
```diff
@@ -151,6 +152,10 @@ func (p *Package) renderOptions(innerPath string, sourceInfo *source.Info, modIn
 		if p.Line == 0 { // invalid Position
 			return ""
 		}
+		// resolve gop_autogen.gop path
```
Comment (xushiwei) [inline internal/godoc/render.go]: `gop_autogen.gop` => `gop_autogen.go`

File: internal/godoc/render.go (hunk 1 above)
Comment (xhyqaq) [reply internal/godoc/render.go]: > don't need check file ext

I don't know if my comment caused the misunderstanding.

If not, what I am parsing here is this line of code
`//line web/filter/details/order.gop:15:1`

If the generated go file is jumped, there will be an extra prefix of the file name.

Comment (xhyqaq) [reply internal/godoc/render.go]: I seem to understand. Lao Xu, what you mean is that you can delete the prefix directly and ignore the extension.

Comment (xushiwei) [reply internal/godoc/render.go]: yes.

**Pull request merged**